Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Dark Government Flavorization

## Changelog:
- Flavorization:
  - Demonic emperors are called Overlords.
  - Demonic kings are called Demon Kings/Queens.
  - Demonic dukes are called Vile Dukes/Duchesses.
  - Demonic counts are called Lords/Ladies.
  - Eldritch kings are called Grand Princes/Princesses.
  - Eldritch dukes are called Princes/Princesses.
  - Eldritch counts are called Lords/Ladies.
- CoAs:
  - Demonic rulers use the Tamgha CoAs.
  - Demonic rulers use dark colors, purple, black, green, yellow, etc.
  - Eldritch rulers use the Tamgha CoAs.
  - Eldritch rulers use dark colors, purple, black, blue, etc.
- Used fitting event backgrounds for the Dark Governments.


## Tests:


# How to test:
Check rulers of Demonic and Eldritch governments.

Play a bit to see some events.
